modin/pandas/test/test_groupby.py

```diff
@@ -1968,6 +1968,21 @@ def compare(obj1, obj2):
     compare(reference_by, result_by)
 
 
+@pytest.mark.parametrize("width", [2**5, 2**10])
+@pytest.mark.parametrize("height", [2**5, 2**10])
+@pytest.mark.parametrize("concat_length", [5, 10])
```

Comment:
Let's get rid of these params and keep the test to a minimal reproducer of the bug.